### DISCUSSION TITLE
My commit changes button is frozen for the past 3 days. What should I do, thanks

### DISCUSSION THREAD

**Discussion by inesreisx** [community/community#58137] [Actions]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/72941372/d79f69a3-943b-4dea-bbb8-1b64ca5bd849)

**Comment by spenserblack** [comment]
This might not be the answer you want :laughing:

I find the GitHub web UI really awkward and cumbersome for making changes. So I'd recommend cloning locally (or working in a codespace!). If you'd rather not learn the Git CLI, there are plenty of text editor plugins to help out. Also apparently the GitHub Desktop app is really helpful, but I haven't used it.

******

For your immediate issue, if it's been frozen for 3 days, it's probably not going to unfreeze. Copy and paste your changes so you have them, and maybe try again.

**Reply by inesreisx** [reply]
hi @spenserblack thanks a lot! hadn't even thought about the app as i prefer to work in-browser or cloud. I'm able to commit and push from github.dev, so I'll be using this method until the button hopefully works again. Again, much appreciated

**Reply by spenserblack** [reply]
If you like github.dev, you'll probably love GitHub Codespaces :slightly_smiling_face:
